cypress/integration/happy-path-with-accessibility.js

```diff
@@ -146,6 +146,24 @@ describe('Get help to retrain smoke test', function() {
     checkAccessibility();
   });
 
+  it('should allow me to get CV advice', function() {
+    cy.go('back')
+    cy.contains('Get CV advice').click()
+    checkAccessibility();
+  });
+
+  it('should allow me to get cover letter advice', function() {
+    cy.go('back')
+    cy.contains('Get cover letter advice').click()
+    checkAccessibility();
+  });
+  
+   it('should allow me to get interview advice', function() {
+    cy.go('back')
```

Comment:
could you add the missing semicolons on each step (I think we missed this on the last pr as well)

Reply:
We don't actually need them in cypress but I've added as per your request